Export rsaverify circuit and Bigint2048

Vishalkulkarni45 opened the pull request:
This PR resolves the issue where rsaverifyCircuit and Bigint2048 couldn't be imported from the mina-attestations package and eliminates the need for manual code copying.

```
import { Bigint2048, rsaVerify65537, rsaSign } from 'mina-attestations/rsa`
```

Vishalkulkarni45 commented:
@mitschabaude can you pls review this PR ?

mitschabaude commented:
Hey @Vishalkulkarni45 thanks, I'll approve this! I have two asks though:
- can you mention `mina-attestations/rsa` in a (very short) section in the README, below the section about `mina-attestations/dynamic` (bottom of the README)
- can you also export `generateRsaKeys65537()` from `utils.ts`? I think this is necessary for testing

Vishalkulkarni45 commented:
@mitschabaude Thanks for the feedback!

- Added a README in `sr/rsa` with brief details on importable functions from `mina-attestations/rsa`.
- Enabled `SHA2` import from `mina-attestations/dynamic`.
- Added a short section on `mina-attestations/rsa` in the main README.

Let me know if you need any changes

mitschabaude commented:
thanks! will merge and release
